internals/cli/cmd_help.go

```diff
@@ -101,11 +102,11 @@ type manfixer struct {
 func (w *manfixer) Write(buf []byte) (int, error) {
 	if !w.done {
 		w.done = true
-		if bytes.HasPrefix(buf, []byte(".TH pebble 1 ")) {
+		if bytes.HasPrefix(buf, []byte(fmt.Sprintf(".TH %s 1 ", cmd.Personality.ProgramName))) {
 			// io.Writer.Write must not modify the buffer, even temporarily
-			n, _ := w.Buffer.Write(buf[:9])
+			n, _ := w.Buffer.Write(buf[:5+len(cmd.Personality.ProgramName)])
 			w.Buffer.Write([]byte{'8'})
-			m, err := w.Buffer.Write(buf[10:])
+			m, err := w.Buffer.Write(buf[6+len(cmd.Personality.ProgramName):])
```

Comment:
The current Pebble on master, according to my manpage, looks like this. So I think this may have been broken?
```
pebbl8(1)                                            General Commands Manual                                           pebbl8(1)

NAME
       pebble - Tool to interact with pebble

SYNOPSIS
       pebble [OPTIONS]
```       
       
I think this intention is to write all the bytes up to before the 1, then replace it with 8, and write the remaining bytes. The first one should have been [:11] and the second [12:]

Reply:
That's interesting because that's not the output I'm getting...
```
pebble(8)                   System Manager's Manual                  pebble(8)

NAME
       pebble - System and service manager
```

```
.TH pebble 8 "22 June 2023"
.SH NAME
pebble \- System and service manager
.SH SYNOPSIS
\fBpebble\fP [OPTIONS]

```

Will investigate, thanks!

Reply:
As a note to myself, using personality here is a bug because the `pebble(1)` string actually comes from the binary name (`go-flags` finds this for us)